prime-router/metadata/fhir_mapping/hl7/codesystem/ExtensionUrlMapping.yml

```diff
@@ -374,4 +374,16 @@
   url: "https://reportstream.cdc.gov/fhir/StructureDefinition/name-suffix-degree"
 # Name suffix professional
 - id: "name-suffix-professional"
-  url: "https://reportstream.cdc.gov/fhir/StructureDefinition/name-suffix-professional"
\ No newline at end of file
+  url: "https://reportstream.cdc.gov/fhir/StructureDefinition/name-suffix-professional"
+# FN.2 Own Surname Prefix
+- id: "humanname-own-surname-prefix"
+  url: "http://hl7.org/fhir/StructureDefinition/humanname-own-name-prefix"
```

Comment:
The correct name is `humanname-own-prefix`, verified by visiting the URL. Not sure how to report the discrepancy in the spreadsheet...

Reply:
good catch, I noticed that too, look like mapping spec authors also make copy-paste errors ;-)